src/js/utils/helpers.js

```diff
@@ -0,0 +1,15 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
```

Comment:
I believe we should check for the screen size as we are doing in other cases.

Sorry we don't have a better documentation around using the Responsive component.

Could you please refer to this file on how to use the Responsive utility from Grommet?

https://github.com/grommet/grommet-qikshot/blob/01929c959caee51e56a496e19bfc5de861946a40/src/js/components/Header.js#L7